content/en/tracing/trace_collection/compatibility/java.md

```diff
@@ -279,6 +279,128 @@ Integrations can be enabled or disabled individually (overriding the default abo
 - Running the Java tracer in Bitbucket is not supported.
 - Loading multiple Java Agents that perform APM/tracing functions is not a recommended or supported configuration.
 
+## GraalVM Native Image support
+
+GraalVM Native Image is a technology that allows you to compile Java applications into native executables. The Datadog Java tracer supports GraalVM Native Image. This allows you to compile your applications into native executables while still benefiting from the tracing capabilities offered by the library.
+
+### Requirements
+
+Use the latest versions of:
+
+- [GraalVM][7]
+- [Datadog Java tracer][1]
+
+### Setup
+
+{{< tabs >}}
+{{% tab "GraalVM" %}}
+To set up the Datadog Java tracer with GraalVM Native Image, follow these steps:
+
+1. Instrument your application, following the steps described on [Tracing Java Applications][6].
+2. When you build a native executable with the `native-image` command, add the following argument:
+   ```shell
+   -J-javaagent:/path/to/dd-java-agent.jar
```

Comment:
This would be clearer with a full command line example:
`native-image -J-javaagent:/path/to/dd-java-agent.jar -jar App.jar`

Reply:
Appreciate the feedback. I'll add the full command line example here. 👍